config/wpcalypso.json

```diff
@@ -26,6 +26,7 @@
 		"dev/test-helper": true,
 		"guided-tours": true,
 		"guided-tours/main": false,
+		"guided-tours/design-showcase-welcome": false,
```

Comment:
This flag, along with the other instances in this PR where the value is false (i.e. all except for development.json) shouldn't be added.